spec/features/idv/hybrid_mobile/hybrid_mobile_spec.rb

```diff
@@ -44,7 +45,11 @@
       # Confirm that jumping to LinkSent page does not cause errors
       visit idv_link_sent_url
       expect(page).to have_current_path(root_url)
-      visit idv_hybrid_mobile_document_capture_url
+
+      # Confirm that we end up on the LN / Mock page even if we try to
+      # go to the Socure one.
+      visit idv_hybrid_mobile_socure_document_capture_url
+      expect(page).to have_current_path(idv_hybrid_mobile_document_capture_url)
```

Comment:
Can we add this quick test to the other document capture feature tests (standard, standard socure, and hybrid socure)?

Reply:
Added to the standard flow feature spec. Will bring up Socure feature specs as a 16th minute today.